heroku/cli
# refactor: migrate pg:psql to oclif/core

[Work item](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bDL5oYAG/view)

## Testing
- checkout this branch and run `yarn` and `yarn build`
- select an app to use that has a database
- run `./bin/run pg:psql --help`
- run `./bin/run pg:psql -a APP_NAME`
    - This should open a connection to the database in your terminal where you can run sql commands
- run `./bin/run pg:psql -a APP_NAME --command "SELECT 1"`
    - This should show the connection and the result of the command and then immediately close the connection.
- Try the commands above with `./bin/run psql` to test the alias
